src/main/java/org/dataone/bookkeeper/resources/UsagesResource.java

```diff
@@ -43,6 +43,7 @@
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.SecurityContext;
 import java.util.*;
```

Comment:
Import individual classes, not packages.  Sometimes IDEA does this automatically behind the scenes without you knowing.